routes/workspace.go

```diff
@@ -131,6 +132,38 @@ func (app *App) UpdateWorkspace(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(updatedWorkspace)
 }
 
+func (app *App) UpdateWorkspaceProps(w http.ResponseWriter, r *http.Request) {
+	workspaceID := mux.Vars(r)["id"]
+
+	if workspaceID == "" {
+		log.Printf("App.UpdateWorkspace - empty workspace id")
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	var updatedProperties model.Attrs
+	reqBody, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Printf("App.UpdateWorkspace - error reading request body %v", err)
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+	err = json.Unmarshal(reqBody, &updatedProperties)
+	if err != nil {
+		log.Printf("App.UpdateWorkspace - error unmarshaling request body %v", err)
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	err = app.store.WorkspaceProvider.UpdateWorkspaceMetadata(workspaceID, &updatedProperties)
+	if err != nil {
+		log.Printf("App.UpdateWorkspace - error updating workspace from provider %v", err)
```

Comment:
```suggestion
		log.Printf("App.UpdateWorkspace - error updating workspace from provider %v", err)
```
log.Printf("App.UpdateWorkspaceMetadata - error updating workspace from provider %v", err)